spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/sql/TestTimestampWithoutZone.java

```diff
@@ -70,7 +70,8 @@
                     "type", "hive",
                     "default-namespace", "default",
                     "parquet-enabled", "true",
-                    "cache-enabled", "false"
+                    "cache-enabled", "false",
+                    "cache.expiration-interval-ms", "0"
```

Comment:
I don't think this needs to change.

Reply:
At one point it was an error if a user specified cache-enabled and had a non-zero cache.expiration-interval-ms.

That's somewhat what this business is about: https://github.com/apache/iceberg/pull/3543#discussion_r766267640

I can remove this though since it's being defaulted to off for the moment anyway.

Reply:
Same as reason mentioned above.